frontend/src/components/regions/UserFeedbackChart/assets.js

```diff
@@ -0,0 +1,114 @@
+// import React, { PureComponent } from 'react';
+import React from "react";
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from "recharts";
+
+const data = [

```

Comment:
Can you create a separate file for this data, this should resemble how we will actually be getting the data from the API , the data will most definitely get here as a json form, however, you can simple create a separate file, and do we know the structure of how we will receive the data?

Reply:
> 

Hi, Viro I made the changes, and also the data const shows us what the structure of the data should more or less look like.